packages/kununu-utils/new-relic/getScript/index.js

```diff
@@ -0,0 +1,199 @@
+/* eslint-disable no-useless-escape */
+const isDisabled = require('../isDisabled');
+
+/**
+ * returns the html script of new relic as needed in html responses
+ *
+ * @param {string} licenseKey
+ * @param {string} applicationId
+ * @return {string}
+ */
+module.exports = (licenseKey, applicationId) => {
+  const newRelicLicenseKey = isDisabled() ? '0000000000' : licenseKey;
```

Comment:
Shouldn't isDisabled just return nothing? Should the script execute at all if it's isDisabled?

Also, why not just check env vars directly here rather than passing them in?